batch/batch/gcp/driver/compute_manager.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any, Dict, Tuple
+import aiohttp
+import logging
+import uuid
+
+from hailtop import aiogoogle
+
+from ...batch_configuration import PROJECT
+from ...worker_config import WorkerConfig
+from ...driver.compute_manager import BaseComputeManager, InstanceDoesNotExist, InstanceState
+
+from .create_instance import create_instance_config
+from .zone_monitor import ZoneMonitor
+from .activity_monitor import ActivityMonitor
+from .disk_monitor import DiskMonitor
+
+log = logging.getLogger('compute_manager')
+
+
+class ComputeManager(BaseComputeManager):
+    def __init__(self, app, machine_name_prefix: str):
+        super().__init__(app, machine_name_prefix)
+        self.credentials = aiogoogle.Credentials.from_file('/gsa-key/key.json')
+        self.compute_client = aiogoogle.ComputeClient(PROJECT, credentials=self.credentials)
+
+        self.zone_monitor = ZoneMonitor(self)
+        self.disk_monitor = DiskMonitor(self)
+        self.activity_monitor = ActivityMonitor(self)
```

Comment:
Yeah this really makes me uncomfortable because my brain is immediately asking: where is the resource cleanup?